composer.json

```diff
@@ -17,17 +17,17 @@
             "homepage": "http://cakephp.nu/"
         },
         {
-            "name":"ADmad",
-            "role":"Contributor",
-            "homepage":"https://github.com/admad"
+            "name": "ADmad",
+            "role": "Contributor",
+            "homepage": "https://github.com/admad"
         }
     ],
     "require": {
-        "cakephp/cakephp": "^3.4"
+        "cakephp/cakephp": "^3.5"
```

Comment:
The changes don't work with 3.4?

Reply:
Sadly getting Errors when running the Testsuite in 3.4.0